cpu/samd21/cmsis/samr21/include/cpu_conf.h

```diff
@@ -20,7 +20,7 @@
 
 #include "cpu_conf_common.h"
 
-#include "samd21.h"
+#include "samr21.h"
```

Comment:
I wanted to avoid too much depth of the folder structure. Therefore I asked for your opinion on (i) moving all vendor headers to *cpu/samd21/include/specific_module_name/* * and (ii) stay with the current location of *cpu/samd21/include/cpu_conf.h*. In that case one has to use `CPU_MODEL` to select the right include file (similar to your solution [here](https://github.com/RIOT-OS/RIOT/pull/5556/files#diff-ba22ede0f27069d40cfaa1d26a79c39cR52)). 

Yes, the *cpu_conf.h* will then contain dedicated `samd21xyz` and `samr21xyz` entries and probably get longer than expected. However, this is a matter of taste and I would like to get multiple opinions on that, especially yours and by other maintainers